src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -329,6 +329,9 @@
         - name: --disable-local-accounts
           type: bool
           short-summary: (Preview) If set to true, getting static credential will be disabled for this cluster.
+        - name: --ossku
+          type: string
+          short-summary: The ossku of the agent node pool.
```

Comment:
can we also provide list of supported values here?

Reply:
Done. Added in a completer too so it can show the options on a double tab when you have --os-sku